backend/app/views/spree/admin/images/index.html.erb

```diff
@@ -4,10 +4,21 @@
 
 <% content_for :page_actions do %>
   <% if can?(:create, Spree::Image) %>
-    <li><%= link_to_with_icon('plus', Spree.t(:new_image), new_admin_product_image_url(@product), :id => 'new_image_link', :class => 'button') %></li>
+    <li><%= link_to_with_icon('plus', Spree.t(:new_image), new_admin_product_image_url(@product), id: 'new_image_link', class: 'button') %></li>
+    <li><%= link_to_with_icon('sort-amount-desc', Spree.t(:variant_image_rules), admin_product_variant_image_rules_url(@product), id: 'variant_image_rules_link', class: 'button') %></li>
   <% end %>
 <% end %>
 
+<div class="deprecation-warning">
+  <div class='warning-title'>
```

Comment:
Prefer double quotes to single quotes for attributes but more importantly prefer consistency.